pkg/kfapp/ossm/ossm_installer.go

```diff
@@ -233,6 +239,27 @@ func (o *OssmInstaller) MigrateDataScienceProjects() error {
 	return result.ErrorOrNil()
 }
 
+func (o *OssmInstaller) ensureServiceMeshInstalled(pluginSpec *ossmplugin.OssmPluginSpec) error {
+	if err := o.VerifyCRDInstalled("maistra.io", "v2", "servicemeshcontrolplanes"); err != nil {
+		log.Info("Failed to find the pre-requisite SMCP CRD, please ensure OSSM operator is installed.")
+		return internalError(err)
+	}
+	status, err := o.CheckSMCPStatus(pluginSpec.Mesh.Name, pluginSpec.Mesh.Namespace)
+	if err != nil {
+		log.Info("An error occurred while checking SMCP status - ensure the SMCP referenced exists.")
+		return internalError(err)
+	}
+	if status != "Ready" {
+		log.Info("The referenced SMCP is not ready.")
+		return internalError(errors.New("SMCP status is not ready"))
```

Comment:
Missed it last time - maybe add smcp details such as ns and name to the log/error msg.

Reply:
added [d47b351](https://github.com/maistra/opendatahub-operator/pull/7/commits/d47b3511b5b91303cf8ed61657da2d791697f462)